applications/hadoop-docker.md

```diff
@@ -0,0 +1,359 @@
+---
+layout: page
+title: "A Production Ops Guide to Deploying Hadoop in Docker Containers"
+keywords: portworx, px-developer, hadoop, yarn, database, cluster, storage
+meta-description: "This Production Ops Guide to running Hadoop in Docker containers will show you how to build a scalable and efficient big data platform on commodity servers. Try it today!"
+sidebar: home_sidebar
+
+---
+
+* TOC
+{:toc}
+
+# Target Audience
```

Comment:
This should be H2, as H1 is the title (and everything below should be a level lower)